Tools/OSD/osd_screen.cs

```diff
@@ -656,20 +665,34 @@ private void rbtSortAlphabetic_CheckedChanged(object sender, EventArgs e)
 
         private void pictureBox1_MouseMove (object sender, MouseEventArgs e) {
 			if(e.Button==System.Windows.Forms.MouseButtons.Left && mousedown==true) {
-				int ansW, ansH;
-				getCharLoc(e.X, e.Y, out ansW, out ansH);
+				int ansW, ansH;
+                int centerPAL = (OSD.SCREEN_H / 2);
+                int centerNTSC = (OSD.SCREEN_H_NTSC / 2);
+                getCharLoc(e.X, e.Y, out ansW, out ansH);
 				//if(ansH >= osd.getCenter() && !(osd.pal_checked() || osd.auto_checked())) {
 					//ansH += 3;
 				//}
 				ansW -= clickX; //запомним куда ткнули
-				ansH -= clickY;
-				                
-				NUM_X.Value = Constrain(ansW, 0, osd.get_basesize().Width - 1);
-                NUM_Y.Value = Constrain(ansH, 0, OSD.SCREEN_H - 1);
+				ansH -= clickY;
 
-                
 
+                if (osd.is_ntsc())
+                {
+                    // started on top screen and went to bottom
+                    if (topScreen && ansH >= centerNTSC)
+                    {
+                        ansH += 3;
+                    }
+                    // started on bottom screen and went to top
+                    else if (!topScreen && ansH <= centerPAL)
+                    {
+                        ansH -= 3;

```

Comment:
did you see everything is inside a if (osd.is_ntsc()) block? What's your suggestion?